server/athenian/api/controllers/settings_controller.py

```diff
@@ -113,10 +113,14 @@ async def set_release_match(request: AthenianWebRequest, body: dict) -> web.Resp
         rule = ReleaseMatchRequest.from_dict(body)
     except ValueError as e:
         raise ResponseError(BadRequestError(str(e))) from e
+    meta_ids = await get_metadata_account_ids(rule.account, request.sdb, request.cache)
     settings = Settings.from_request(request, rule.account)
+    prefixer = await Prefixer.load(meta_ids, request.mdb, request.cache)
+
     match = ReleaseMatch[rule.match]
```

Comment:
(noticed while reading the code) What happens if we send a broken match name, e.g. `xxx` :thinking:

Reply:
Handled by especifico through jsonschema validation I think, already tested:
https://github.com/athenianco/athenian-api/blob/master/server/tests/controllers/test_settings_controller.py#L161

Reply:
This is included in the unit tests, right?

Reply:
Yes, the "whatever" here is the invalid match name
https://github.com/athenianco/athenian-api/blob/master/server/tests/controllers/test_settings_controller.py#L161